src/folder/schema.ts

```
@@ -0,0 +1,147 @@
+  minimumPledge: monetaryString(), // Using custom transformer
+  currency: z.enum(currencyEnum.enumValues).optional(),
+  endDate: z.date().refine((date) => date > new Date(), { message: "End date must be in the future" }),
+  publisherId: z.string().uuid().optional().nullable(),
+  categoryId: z.string().uuid().optional().nullable(),
+  publisherType: z.enum(publisherTypeEnum.enumValues).default('user'),
+  featuredImage: z.string().url().optional().nullable(),
+  risks: z.string().optional().nullable(),
+  faq: z.array(z.object({
+    question: z.string().min(1),
+    answer: z.string().min(1)
+  })).optional().nullable(),
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)